_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -3317,7 +3317,9 @@ mfxStatus VideoVPPHW::PreWorkInputSurface(std::vector<ExtSurface> & surfQueue)
                     dstTempSurface.Data.MemId = &dstHandle;
 
                     mfxI64 verticalPitch = (mfxI64)(srcTempSurface.Data.UV - srcTempSurface.Data.Y);
-                    verticalPitch = (verticalPitch % srcTempSurface.Data.Pitch)? 0 : verticalPitch / srcTempSurface.Data.Pitch;
+                    // offset beetween Y and UV must be align to pitch
```

Comment:
Do we have such requirement in our docs?

Reply:
I think it's a subject for release notes..

Reply:
Is that an actual requirement or CM feature request?

Reply:
I think this is feature request, but as mentioned in issue discussion, re-writing of cm kernel will require significant efforts.